tsconfig.build.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
It is a bit cumbersome to manage references everywhere, but I'd recommend basing the `build` config on the non-build config and only make it actually build. E.g. like this.

```jsonc
{
	"extends": "./tsconfig.json",
	// project references here.
	"compilerOptions": {
		"rootDir": "src",
		"outDir": "build"
	},
	"include": ["src/**/*.ts"],
	"exclude": [ /* test files, etc */ ]
}
```

Reply:
Upside of this, the editor experience is using the same settings, so no surprises at build time